sdcm/cluster.py

```diff
@@ -2567,29 +2568,20 @@ def restart_node_with_resharding(self, murmur3_partitioner_ignore_msb_bits: int
         # Change murmur3_partitioner_ignore_msb_bits parameter to cause resharding.
         with self.remote_scylla_yaml() as scylla_yml:
             scylla_yml.murmur3_partitioner_ignore_msb_bits = murmur3_partitioner_ignore_msb_bits
-
-        search_reshard_start = self.follow_system_log(patterns=[DB_LOG_PATTERN_RESHARDING_START])
-        search_reshard_finish = self.follow_system_log(patterns=[DB_LOG_PATTERN_RESHARDING_FINISH])
         start_scylla_timeout = 7200
-        self.start_scylla(timeout=start_scylla_timeout)
-        resharding_started = list(search_reshard_start)
-        resharding_finished = list(search_reshard_finish)
-
-        if resharding_started:
-            self.log.debug(f'Resharding has been started successfully '
-                           f'(murmur3_partitioner_ignore_msb_bits={murmur3_partitioner_ignore_msb_bits})')
-        else:
-            raise Exception(f'Resharding has not been started '
-                            f'(murmur3_partitioner_ignore_msb_bits={murmur3_partitioner_ignore_msb_bits}) '
-                            'Check the log for the details')
-
-        if resharding_finished:
-            self.log.debug(f'Resharding has been finished successfully '
-                           f'(murmur3_partitioner_ignore_msb_bits={murmur3_partitioner_ignore_msb_bits})')
-        else:
-            raise Exception(f'Resharding has not been finished within {start_scylla_timeout}'
-                            f'(murmur3_partitioner_ignore_msb_bits={murmur3_partitioner_ignore_msb_bits}) '
-                            'Check the log for the details')
+        resharding_not_started_msg = f'Resharding has not been started \
+        (murmur3_partitioner_ignore_msb_bits={murmur3_partitioner_ignore_msb_bits}) Check the log for the details'
```

Comment:
not sure how those lines are better than a massage having the regular expression that didn't matched the logs

you can catch the exception, and append more context info if needed, changing the exception message in this case, make it a bit harder to debug.

Reply:
yes, I was thinking more about appending the context.
See this msg misses info about exact pattern (and sometimes just Scylla changes log structure) and missing node affected.